project/bin/{{shellm_package_script_name}}.jinja

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+## \brief {{ project_description }}
+## \usage {{ shell_package_script_name }} [-h]
```

Comment:
Often times I think I remember that something is a convention in the UNIX/POSIX world, such as using only short options in usage messages... but I actually can't prove it :sweat_smile: And even the most common commands do not all follow that convention. So, I don't have any strong opinions here. I personally prefer using only short names because it's easier to maintain, and produces smaller lines when written to the console, preventing possible hard-wraps.

Reply:
Sure. Just `-h` works. 👍🏻